chain/yoda/execute.go

```diff
@@ -40,14 +64,15 @@ func SubmitReport(c *Context, l *Logger, key keys.Info, id otypes.RequestID, rep
 
 		txBldr := auth.NewTxBuilder(
 			auth.DefaultTxEncoder(cdc), acc.GetAccountNumber(), acc.GetSequence(),
-			200000, 1, false, cfg.ChainID, fmt.Sprintf("yoda:%s/exec:%s", version.Version, execVersion), sdk.NewCoins(), c.gasPrices,
+			200000, 1, false, cfg.ChainID, fmt.Sprintf("yoda:%s/exec:%s", version.Version, executorVersion), sdk.NewCoins(), c.gasPrices,
```

Comment:
Let's do some more experiment regarding gas. I'm not comfortable hardcoding 200k.